src/as-content-rating.c

```diff
@@ -1576,3 +1579,59 @@ as_content_rating_new (void)
 	content_rating = g_object_new (AS_TYPE_CONTENT_RATING, NULL);
 	return AS_CONTENT_RATING (content_rating);
 }
+
+/**
+ * as_oars_version_from_string:
+ * @value: the string.
+ *
+ * Converts the text representation to an enumerated value.
+ *
+ * Returns: a #AsOarsVersion or %AS_OARS_VERSION_UNKNWON for unknown
+ *
+ * Since: 1.0
```

Comment:
Please drop the "Since" line, since this is internal API now (and it will make backporting the patch easier).